### ISSUE TITLE
IsBorderRequired not effective in Win10

### ISSUE THREAD

**Issue by eamon-cai** [lovettchris/wincam#2]
IsBorderRequired can be used to remove the yellow border during capture, but it seems to only work on Win11. The method to remove it on Win10 cannot be found. However, some discussions have been found: https://github.com/microsoft/Windows.UI.Composition-Win32-Samples/issues/97#issuecomment-1013443790

**Comment by lovettchris** [comment]
Thanks for the tip, I implemented this in version 0.0.5.

**Comment by lovettchris** [comment]
Which you can see in [this commit ](https://github.com/lovettchris/wincam/commit/abe940ab50fcb070672d5c452b01d1062d5e613c) is easier to do in C++:

```cpp
    auto session3 = m_session.as<winrt::Windows::Graphics::Capture::IGraphicsCaptureSession3>();
    if (session3 != nullptr) {
        session3.IsBorderRequired(false);
    }
```

You can also see I added a flag on start capture to also make captureCursor optional.
